### ISSUE TITLE
Another example of use? Feedback more than Issue

### ISSUE THREAD

**Issue by Mnark** [forty2/tivo-remote#1]
It took me quite a while to figure out how to use this module. 

It appears to export a property called 'default' rather than the discovery object. 

Is there a reason for not just exporting "discovery" direct;y from index.js ("exports = discovery;" rather than "exports.default = discovery;")?  

I ended up using.. 

 var discovery = require('tivo-remote').default
  discovery.on('founddevice', (device) => {
      console.log(`Found a device: ${device.name} (${device.ip})`);
  })
  discovery.on('lostdevice', (device) => {
      console.log(`Lost a device: ${device.name} (${device.ip})`);
  })
  discovery.discover();

**Comment by forty2** [comment]
Sorry for the trouble!

The reason this happens is because the module is written with ES2015+ syntax and transpiled (using Babel).  When you mark something as the default export in an ES6 module, the transpiled version will have
```javascript
exports.default = <whatever>
```
because of the semantics of ES6 modules (specifically because ```export default foo``` and ```export { foo as default }``` are specified to mean the same thing).

Babel also defines a property on the ```exports``` object called ```__esModule``` with a value of ```true```, which I believed was enough for Node to transparently handle this situation when calling ```require()``` on an ES6 module, but it would appear that that is _not_ the case.

I've spent some quality time with Google trying to work out how other modules deal with this situation, and what I've learned so far is that this is apparently a massive can of worms that I was previously blissfully unaware of just because I'm addicted to [object rest properties](https://github.com/tc39/proposal-object-rest-spread) and the [function bind operator](https://github.com/tc39/proposal-bind-operator) (which currently require transpilation because they're not part of the standard yet).

So, the bottom line: something needs to be done, because ```require('tivo-remote')``` should work even if you're not using a transpiler.  But I'm not sure _what_ needs to be done, because ```import Discovery from 'tivo-remote'``` should work if you _are_ using a transpiler.  I'm leaving this open until I figure out what the "best practice" is.  Until then, the quickest path to joy is probably either to use ```var Discovery = require('tivo-remote').default;``` or, if it's useful for other reasons too, use a transpiler.
